src/App.tsx

```diff
@@ -52,7 +59,7 @@ function App(props: RouteComponentProps) {
     const VIS_PADDING = {
         top: isMinimalMode ? 0 : 60,
         right: isMinimalMode ? 0 : 60,
-        bottom: isMinimalMode ? 0 : 60,
+        bottom: 0,
```

Comment:
It looks like now there is no 'dotted' texture on the bottom of the Chromoscope that we used to see before. Would this be due to the change made to this line? I wasn't sure why the conditional operation with the `isMinimalMode` had to be removed here.

![Screenshot 2024-08-05 at 4 58 02 PM](https://github.com/user-attachments/assets/cf84d8b0-2c57-4fc1-80d6-a5aae33eaa49)